<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Plant Note</string>
  </property>
  <widget class="QTextEdit" name="textEdit_note">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>281</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_descripe_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter a note for this plant</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(94, 170, 142);
border: none;
color: white;</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
